src/core/Akka/Event/StandardOutLogger.cs

```diff
@@ -138,15 +138,15 @@ public static void PrintLogEvent(LogEvent logEvent)
             catch (FormatException)
             {
                 /*
-                 * If we've reached this point, the `logEvent` itself is informatted incorrectly. 
+                 * If we've reached this point, the `logEvent` itself is formatted incorrectly. 
                  * Therefore we have to treat the data inside the `logEvent` as suspicious and avoid throwing
                  * a second FormatException.
                  */
                 var sb = new StringBuilder();
                 sb.AppendFormat("[ERROR][{0}][Thread {1}][StandardOutLogger] ", logEvent.Timestamp, 0);
-                sb.AppendFormat("Encoutered System.FormatException while recording log: [" +
+                sb.AppendFormat("Encountered System.FormatException while recording log: [" +
                                 logEvent.LogLevel().PrettyNameFor() + "]")
-                    .AppendFormat("[" + logEvent.LogSource + "][" + logEvent.Message + "]");
+                    .Append("[" + logEvent.LogSource + "][" + logEvent.Message + "]");
```

Comment:
I thought this was the fix, but sadly no

Reply:
`logEvent.Message` is the culprit